test/unit/Cover/burnStake.js

```diff
@@ -167,31 +160,28 @@ describe('burnStake', function () {
     const fixture = await loadFixture(setup);
     const { cover, stakingProducts } = fixture;
     const [internal] = fixture.accounts.internalContracts;
-    const { productId, coverAsset, period, amount, targetPriceRatio } = coverBuyFixture;
-    const { segmentId, coverId: expectedCoverId } = await buyCoverOnOnePool.call(fixture, coverBuyFixture);
+    const { productId, coverAsset, period, amount } = coverBuyFixture;
+    const { coverId: expectedCoverId } = await buyCoverOnOnePool.call(fixture, coverBuyFixture);
 
     // TODO: need to figure out a way to change the capacity ratio here
-    // ...
+    // ...   currently, this can can be done only in an integration test
```

Comment:
nah, not really, needs a copy of the contract which is messy. best to remove the todo entirely, this is tested in other unit tests (stakingpool for example).